manuscripts.json

```diff
@@ -1,5 +1,12 @@
 {
   "preprints": {
+    "88058": {
+      "preprintDoi": "88058",
+      "msas": [
+        "Biochemistry and Chemical Biology",
+        "Structural Biology and Molecular Biophysics"
+      ]
+    },
```

Comment:
@fred-atherden not sure how this happened?! Let's have a chat to see if we can get to the bottom of it.